server/src/middlewares/auth.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import user from '../models';
+
+/**
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+export function authMiddleware(req, res, next) {
+    try {
+        const accessToken = req.cookies['access-token']
+        console.log(accessToken);
+
+        const payload = jwt.verify(accessToken, process.env.SECRET_TOKEN)
+        console.log(payload);
+        req.user = { id: payload._id };
```

Comment:
In line 19 we are overriding req.user with whole model so we can safely remove this line.